.themes/default/source/arcs.html

```diff
@@ -15,8 +15,6 @@ <h3 class="arc-title"><a name="{{ rp.name | slugify }}" href="#{{ rp.name | slug
       {% include rp.html rp=arc_rp %}
     {% endfor %}
     </ul>
-  {% else %}
-    {% include rp.html rp=rp %}
```

Comment:
Well, it's more of a display that displays arcs when they exist, and also other ones. I dunno, haven't really thought about it.

I'd rather not change it now as part of this, that should be its own thing.